pkgs/servers/sql/postgresql/ext/pgjwt.nix

```diff
@@ -1,14 +1,14 @@
-{ lib, stdenv, fetchFromGitHub, postgresql }:
+{ lib, stdenv, fetchFromGitHub, postgresql, unstableGitUpdater }:
 
 stdenv.mkDerivation {
   pname = "pgjwt";
-  version = "unstable-2021-11-13";
+  version = "unstable-2023-03-02";
```

Comment:
should we actually do two versions og pgjwt, stable and unstable like a lot of other releases do?